packages/app-council/src/Motions/Motion.tsx

```diff
@@ -0,0 +1,172 @@
+// Copyright 2017-2019 @polkadot/app-democracy authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { I18nProps } from '@polkadot/ui-app/types';
+import { RawParam } from '@polkadot/ui-params/types';
+import { Option, Proposal, Votes } from '@polkadot/types';
+
+import BN from 'bn.js';
+import React from 'react';
+import styled from 'styled-components';
+import { ActionItem, Chart, Voting } from '@polkadot/ui-app';
+import VoteThreshold from '@polkadot/ui-params/Param/VoteThreshold';
+import { withCalls, withMulti } from '@polkadot/ui-api';
+import settings from '@polkadot/ui-settings';
+import { formatNumber } from '@polkadot/util';
+
+import translate from '../translate';
+
+const COLORS_YAY = settings.uiTheme === 'substrate'
```

Comment:
Not crazy about this duplication at all. We should rather have it external and shared between the apps using it.